text/0010-gatsby-docs-localization.md

```diff
@@ -0,0 +1,212 @@
+
+There are other SaaS platforms such as Transifex that fix some of Crowdin's issues. However, the core issue relies in that they are new systems that most developers are unfamiliar with. We are relying on developers from the Gatsby community rather than professional translators, and those developers tend to be more familiar with GitHub's UI.
+
+### Other GitHub-based arrangements
+
+The monorepo isn't the only way to organize our code. We could do something like React and have each language be in a separate repo copy. This makes it easier to copy content and check for changes, since we can just use `git merge`.
+
+But this has its own drawbacks as well. Each language is its own website and we can't easily switch between languages. The rest of Gatsby's code is in a monorepo and has tooling to make it work, so it makes sense to continue with that paradigm. We'd also like to dogfood Gatsby's localization utilities such as [`gatsby-plugin-i18n`](https://www.gatsbyjs.org/docs/localization-i18n/#gatsby-plugin-i18n).
+
+## Adoption Strategy
+
+To test-drive the process, we can start with three languages or so, based on user interest in this RFC. We will have to manually set up those translations while we create the automation, but they should provide a test bed to make sure our checks work.
+
+The only thing we need to do before those languages are set up is move content over to `/docs/en` and handle the routing appropriately. Once that's done, the translations can start and we can divide up the other changes needed for the website.
+
+Once the automation work for setting up new translations is done, we can open up pull requests to create new language to more people.
+
+When the first languages have a "core" set of pages translated (for example, the Home Page, Tutorial, and API Reference), that translation can be marked as "Complete".
+
+
+## How will we teach this
+
+We'll create additional documentation in both the monorepo and the site itself on the process of contributing translations. There should be documentation for:
+
+* How to contribute a new translation (with a link to the progress issue)
+* How the automation works and what to expect
+* The responsibilities of a translation maintainer
+* Tips for translators
+
+We should also do outreach to Gatsby users and communities around the world to inform them that the translations are in progress and (once completed) spread the word that the localizations exist.
+
+
+## Unresolved Questions
+
+* If we intend to publish different versions of the Gatsby docs, how will versioning work across the different languages?
+* How do we limit the Algolia search to only return results for each language?
+* How do we manage SEO?
+* What is the process for determining who gets to maintain a language? For React, we pretty much let the first two people to create a pull request start their own language translation.
```

Comment:
@tesseralis This one is tricky 😅 If I remember right for React there were some languages that had a drop off from the maintainers along the way and it's difficult. The easiest way to at least point to some folks that are willing to maintain the docs, is to ask people that translated other resources like this before, from example React, we have a lot of experience with this focus, and as I said before  many folks would be interested in jumping here to translate too ❤️ 

At least we can pull off one maintainer from the previous React docs to jump in here and give some direction to the translations and to organize people, we already have a lot of gotchas, wikis, and strategies to work already built so it'd be easier and faster to start with for example 🤔